<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propcomponentdlg</class>
 <widget class="QDialog" name="propcomponentdlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component Property</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName">
       <property name="maxLength">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Reference</string>
       </property>
       <property name="buddy">
        <cstring>editRef</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editRef">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32767</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QGridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="editHeight">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel4">
           <property name="text">
            <string>Height</string>
           </property>
           <property name="buddy">
            <cstring>editHeight</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>Width</string>
           </property>
           <property name="buddy">
            <cstring>editWidth</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editWidth">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel5">
           <property name="text">
            <string>Block</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editBlock">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkUsePtn">
         <property name="text">
          <string>Use Pattern</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkNotUseBitPtn">
         <property name="text">
          <string>Not Use Bit Pattern</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPushButton" name="buttonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editRef</tabstop>
  <tabstop>editWidth</tabstop>
  <tabstop>editHeight</tabstop>
  <tabstop>editBlock</tabstop>
  <tabstop>checkUsePtn</tabstop>
  <tabstop>checkNotUseBitPtn</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>propcomponentdlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
